airbyte-integrations/connectors/source-kafka/src/main/resources/spec.json

```diff
@@ -78,6 +78,12 @@
         "type": "integer",
         "default": 500
       },
+      "polling_time": {
+        "title": "Polling Time",
+        "description": "Amount of time Kafka connector should try to poll for messages.",
+        "type": "integer",
+        "default": 5000
+      },
```

Comment:
@misteryeo can you check this change in the specification of Kafka connector. This is something we want to expose to users?